lib/Basic/FileTypes.cpp

```diff
@@ -147,6 +149,7 @@ bool file_types::isAfterLLVM(ID Id) {
 bool file_types::isPartOfSwiftCompilation(ID Id) {
   switch (Id) {
   case file_types::TY_Swift:
+  case file_types::TY_ASTDump:
```

Comment:
This one's about inputs rather than outputs, so I'd prefer it go in the "false" section. It's not likely to come up, though.